src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -999,61 +999,61 @@ static T Vectorized128(ref T xRef, nuint remainder)
                 switch (blocks)
                 {
                     case 7:
-                    {
-                        Vector128<T> vector = TTransformOperator.Invoke(Vector128.LoadUnsafe(ref xRef, remainder - (uint)(Vector128<T>.Count * 7)));
-                        vresult = TAggregationOperator.Invoke(vresult, vector);
-                        goto case 6;
-                    }
+                        {
```

Comment:
Looks like there's a very large amount of whitespace diff here, and not for the better.

I think this is the legacy quirk where `case` defaults to being indented (which is good) but doesn't factor in when that is a block (you have to explicitly opt-in to `csharp_indent_case_contents_when_block = false` to get it to format more cleanly).